content/attacks/posts/2019-03-25-CoinBene.md

```diff
@@ -0,0 +1,45 @@
+---
+date: 2019-03-25
+target-entities: CoinBene
+entity-types:
+- Custodian
+- Exchange
+attack-types: Wallet Hack
+title: "CoinBene Exchange Allegedly Loses Over $100 Million in Suspected Hack"
+---
+
+## Summary
+
+CoinBene, a Singapore-based cryptocurrency exchange, reportedly suffered a security breach on March 25, 2019, resulting in the loss of over $105 million worth of various tokens and coins. The exchange [denied being hacked](https://cointelegraph.com/news/over-100-million-missing-coinbene-claims-maintenance-a-month-of-questions-point-toward-a-hack) and claimed it was undergoing maintenance. CoinBene has not provided any official statement regarding the incident.
+
+## Attackers
+
+The identity of the attacker remains unknown. The following addresses were involved in the hack:
+
+- [0xA1BF1ED1e8DE34477fB3dCE27c2eA2EA4163AcbA](https://etherscan.io/address/0xa1bf1ed1e8de34477fb3dce27c2ea2ea4163acba)
+- [0xb3dF999C5dC026dea265AEB02b8519844C9b6D5E](https://etherscan.io/address/0xb3dF999C5dC026dea265AEB02b8519844C9b6D5E)
+- [0xba351e7f0c630b3baa30a0ff38f6f4a333ef2133](https://etherscan.io/address/0xba351e7f0c630b3baa30a0ff38f6f4a333ef2133)
+
+## Losses
+
+CoinBene lost $105,150,329 worth of various assets. The most valuable [stolen assets breakdown](https://github.com/elementus-io/coinbene-analysis#market-value-of-assets-withdrawn):
```

Comment:
The second sentence here is a little awkward.  Suggest: "Breakdown of top 5 stolen assets by value:"